staging/src/k8s.io/kubectl/pkg/kuberc/kuberc_test.go

```diff
@@ -1160,12 +1160,13 @@ func TestApplOverrideBool(t *testing.T) {
 			if err != nil {
 				t.Fatalf("unable to find the command %v\n", err)
 			}
-
 			err = actualCmd.ParseFlags(test.args[1:])
 			if err != nil {
 				t.Fatalf("unexpected error %v\n", err)
 			}
-
+			if _, ok := actualCmd.Annotations[KubeRCOriginalCommandAnnotation]; !ok {
```

Comment:
Can you actually check the value of the annotation, does it match the expectations? Not only its presence.